### ISSUE TITLE
OpenCTI and TheHive

### ISSUE THREAD

**Issue by khoabui333** [LuatixHQ/opencti#4766]
Hi, I connected OpenCTI with TheHive by TheHive connector. Then, I analyze a case in TheHive by using Cortex and it returned me a report. However, my OpenCTI can only ingest the case from TheHive and the report cannot be ingested with the case.

![image](https://github.com/OpenCTI-Platform/opencti/assets/32787602/ea6ef31f-95de-43a6-b502-2af5933848da)

Is there any method that can help me to bring the report from TheHive to OpenCTI.
Thanks.

**Comment by cmandich** [comment]
@khoabui333 , this is an interesting feature request. I believe the current connector only imports alerts and cases. 

Just to confirm, is this the connector you are referencing? 
https://github.com/OpenCTI-Platform/connectors/tree/master/external-import/thehive

**Comment by khoabui333** [comment]
yes, I already used it but it didn't allow me to import report to OpenCTI

**Comment by nino-filigran** [comment]
Thanks for your request. Indeed, the Hive only imports Cases (and not reports) and there's currently no way to import directly from the hive using the connector. 
Is there q specific reason why you would like to import reports from the Hive?

**Comment by alice-debra** [comment]
Hello @khoabui333,
Due to a lack of recent activity on this issue, we will proceed to close it for now. We sincerely thank you for your feedback. Please don’t hesitate to reopen the issue if you still have any questions.
Kind regards,
Alice
